pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -684,3 +752,45 @@ def test_link_object_of_complex_object_should_also_be_protected_from_deletion(
                     self.shell,
                     self.cluster.default_rpc_endpoint,
                 )
+
+    @allure.title("Locked object should be protected from deletion after the storage nodes are restarted")
+    @pytest.mark.parametrize(
+        # Only complex objects are required for this test
+        "new_locked_storage_object",
+        [pytest.lazy_fixture("complex_object_size")],
+        indirect=True,
+    )
+    def test_the_object_lock_should_be_kept_after_restarting_the_nodes(
+            self,
+            new_locked_storage_object: StorageObjectInfo,
+            enable_metabase_resync_on_start,
+    ):
+        """
+        Lock objects should fill metabase on resync_metabase
+        """
+        with allure.step(f"Delete metabase files from storage nodes"):
```

Comment:
IIUC this should be done after the initial deletion attempt. Likely that's your problem with `deadline exceeded`.

Reply:
> IIUC this should be done after the initial deletion attempt. Likely that's your problem with `deadline exceeded`.

I added an attempt to delete the object before deleting the metabase.
The result is the same:
https://http.t5.fs.neo.org/86C4P6uJC7gb5n3KkwEGpXRfdczubXyRNW5N9KeJRW73/339-1694555644/index.html#suites/b34feeefa7bd0ee424f598738938f10a/3a36ea27b17b99e8/

Reply:
@roman-khimov @carpawell  please take a look at the test result.
https://http.t5.fs.neo.org/86C4P6uJC7gb5n3KkwEGpXRfdczubXyRNW5N9KeJRW73/339-1694717358/index.html#suites/b34feeefa7bd0ee424f598738938f10a/b60696c863dd0db8/

It seems that I'm deleting the metabase correctly - removing storage/meta0 and meta1:
https://github.com/nspcc-dev/neofs-testlib/blob/master/src/neofs_testlib/hosting/docker_host.py#L137

Is it possible that this is a bug?

Reply:
Fixed